syne_tune/optimizer/baselines.py

```diff
@@ -778,6 +779,50 @@ def __init__(
         )
 
 
+class NSGA2(FIFOScheduler):
+    """
+
+    See :class:`~syne_tune.optimizer.schedulers.searchers.RandomSearcher`
+    for ``kwargs["search_options"]`` parameters.
+
+    :param config_space: Configuration space for evaluation function
+    :param metric: Name of metric to optimize
+    :param population_size: See
```

Comment:
Can you add a new comment here to describe the population? We probably dont want to cross reference in this case as those are different methods which my also have different defaults. 

Also a minor point: I do not think we need to put default in the comment (it adds a risk to have them async with the code).